### ISSUE TITLE
A few users who open H4CC menu seem to think that all it has are forms, deadlines, trainings

### ISSUE THREAD

**Issue by nickykrause** [fecgov/fec-testing#67]
During multiple recent rounds of usability testing, users were asked the following question:

> You work for a corporation that is interested in raising money to support a candidate. Can you find any information on this website to help get you started?

Typically, the users who are asked this question did _not_ find the answer in the `Help for candidates and committees` menu, but rather through other means. For example, here are the results from 2 users:

> The first user succeeded fairly quickly by scrolling to the H4CC section of the homepage and clicking the side navigation for "corporations." The other user searched "corporations" in the legal resources main search and was a bit disappointed with the results, which he described as "spammy." 

In both of these cases, users actually had opened the H4CC menu, but it seemed as if they didn't see anything that "jumped out" as relevant, and therefore opted for other paths.

A similar thing has occurred for this question:

> You work for a candidate's authorized committee and have received a donation from a foreign national. You want to know if you can accept that donation or not. Can you find the answer to that question using this website?

In this case, _many_ users did go to H4CC and successfully click on the `Get started` button to access the H4CC landing page. However, there were a small proportion of users who went to H4CC and seemed to miss the button, and ended up at legal resources instead (which is possibly fine?)

In summary: Although there have been _many_ successes with the H4CC menu, especially with the latter question, there is a small group of people who don't seem to understand that H4CC contains more than what is listed in the menu, and who may be missing useful information.
